simplekv/__init__.py

```diff
@@ -186,7 +186,7 @@ def _check_valid_key(self, key):
 
         :param key: The key to be checked
         """
-        if not isinstance(key, text_type):
+        if not isinstance(key, key_type):
             raise ValueError('%r is not a unicode string' % key)
```

Comment:
"unicode string" is misleading here then? Should probably be 'valid key type' or `str(key_type)` ?